catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -169,6 +169,7 @@
 
     <feature name="abdera" install="manual" version="${project.version}"
              description="Apache Abdera">
+        <feature prerequisite="true">wrap</feature>
```

Comment:
When you make a PR for removing prereqs, try and find a better place to start this